charts/studio/templates/deployment-studio-backend.yaml

```
@@ -127,13 +132,55 @@ spec:
             - secretRef:
                 name: {{ .Values.studioBackend.envFromSecret }}
             {{- end }}
+        {{- if and .Values.global.basePath (not (eq .Values.global.basePath "/")) }}
+        - name: nginx-sidecar
+          image: "{{ .Values.studioBlobvault.image.repository }}:{{ .Values.studioBlobvault.image.tag | default "1.25.1-alpine" }}"
```

```suggestion
          image: "{{ .Values.studioBlobvault.image.repository }}:{{ .Values.studioBlobvault.image.tag }"
```
I think we should unset default here since we have it in values.yaml already